### ISSUE TITLE
Proper syntax for camera_info_url paramaeter

### ISSUE THREAD

**Issue by pitosalas** [UbiquityRobotics/raspicam_node#107]
I am using this in my .launch file

    <param name="camera_info_url" value="package://gpg_bran/camera_info/camerav2_1280x960.yaml"/>

And yet I am getting this error message:

[ INFO] [1603729230.468427408]: Unable to open camera calibration file [/home/pi/.ros/camera_info/camerav2_1280x960.yaml]
[ WARN] [1603729230.468570167]: Camera calibration file /home/pi/.ros/camera_info/camerav2_1280x960.yaml not found.

Can you clarify how the value= for this param is interpreted? Thanks!

**Comment by MoffKalast** [comment]
Yeah I think I've had a similar issue in the past, something isn't set up right in the pi images for the package:// syntax resolution. The file:/// syntax does work but you have to specify the entire path.

**Comment by pitosalas** [comment]
Thanks. I wish there was a tad more documentation. For example the node looks for a "default" camera_info file and a "specific" one. Not sure why. Thanks though I've gotten it to work.

Pito Salas
Faculty, Computer Science
Brandeis University
